app/controllers/case_logs_controller.rb

```diff
@@ -12,6 +12,12 @@ def index
 
     @pagy, @case_logs = pagy(filtered_case_logs(current_user.case_logs))
 
+    search_param = params["search-field"]
```

Comment:
If you use the generic component this is just `search` not `search-field` I think